data/test/scenarios/test_terrain_mask.cfg

```diff
@@ -141,6 +274,15 @@ Hh, Hh"#enddef
 )}
 
 # mask_file= works (this is the same as the above test in all other respects)
+#####
+# API(s) being tested: [terrain_mask][rule]
+##
+# Actions:
+# Use a terrain mask rule to change existing terrain to either Ko or Co.
```

Comment:
This one's missing from `wml_test_schedule`, and its name expands to `test_terrain_mask_file`.

These docs are wrong (based on running the test in interactive mode), it's also adding hills.